insights/parsers/hammer_compute_resource_list.py

```diff
@@ -0,0 +1,56 @@
+"""
+HammerComputeResourceList - command ``hammer --interactive 0 --output json compute-resource list``
+===================================================
```

Comment:
Better keep the title and the `==` lines in the same length. Otherwise, errors will report when building the Document for it.

You can also check the doc in your local by:

~~~
# cd insights-core
# (cd docs && make html)
OR
# (cd docs && sphinx-build -b html -d _build/doctrees  -W -j 8 . _build/html)
~~~